microcosm_postgres/factories.py

```diff
@@ -25,6 +25,12 @@
     max_overflow=10,
     # echo all SQL
     echo=False,
+    # always use SSL to connect to postgres
+    require_ssl=True,
+    # verify SSL certificate
+    verify_ssl=False,
+    # specify certificate path
+    ssl_cert_path="/path/to/my/pem",
```

Comment:
Is that overridden by something?

Reply:
This location of the actual cert should be passed into the configuration.

Reply:
In that case, I suggest not having a default, and raising an exception if verify_ssl is on but the location has not been passed in.